plugins/wazo-yealink/v86/plugin-info

```diff
@@ -1,5 +1,5 @@
 {
-    "version": "1.0.9",
+    "version": "1.1.1",
     "description": "Plugin for Yealink for CP920, T27G, T3X, T4X and T5X series in version 86.",
```

Comment:
Should we add  a new key `"python3_support": true` to the plugin-info to be able to display if a plugin is compatible or not when we release a version of provd in python3?

Reply:
No, we will use a different URL for the python3 plugins.